collections/hub/other_tests/bulk_transfers/bulk-transfer-timeout-scenario.json

```diff
@@ -301,18 +302,63 @@
                 "id": 1,
                 "description": "Status code is 200",
                 "exec": [
-                  "expect(response.status).to.equal(200)"
+                  "expect(environment.participantsPositionResponse.code).to.equal(200)"
                 ]
               },
               {
                 "id": 2,
                 "description": "Payerfsp position after (bulk) timeout should be same as position before prepare.",
                 "exec": [
-                  "expect(response.body[0].value).to.equal(+environment.payerfspPositionBeforePrepare)"
+                  "expect(environment.participantsPositionResponse.body[0].value).to.equal(+environment.payerfspPositionBeforePrepare)"
                 ]
               }
             ]
-          }
+          },
+          "scripts": {
+            "postRequest": {
+              "exec": [
+                " const expectedStatusCode = 200;",
+                " environment.participantsPositionResponse = null",
+                " ",
+                " // lets get retry config",
+                " const maxRetryCount = Number.parseFloat(environment.RETRY_MAX_ATTEMPTS) || 20",
+                " const maxRetryWait = Number.parseFloat(environment.RETRY_MAX_WAIT_IN_MS) || 250",
+                " ",
+                " async function main() {",
+                "   let _response = response",
+                " ",
+                "   for (let retryCount = 0 ; retryCount < maxRetryCount; retryCount++) {",
+                "     console.log(`retry-count=${retryCount+1}, max-count=${maxRetryCount}, max-wait=${maxRetryWait}`)",
+                "     ",
+                "     const isOk = _response.code === expectedStatusCode && _response.body[0].value === (+environment.payerfspPositionBeforePrepare)",
+                "     ",
+                "     if (isOk) {",
+                "       environment.participantsPositionResponse = _response",
```

Comment:
If you don't need to access these variables in other requests, you can use requestVariables.